### ISSUE TITLE
No EC in my TPM chip 

### ISSUE THREAD

**Issue by jemm14** [opencit/opencit#11]
I get the follwing trying to install the trust agent:

`Error: Error getting EC: TpmModule.setCredential returned nonzero error (19) Error: Error provisioning TPM: TpmModule.setCredential returned nonzero error (19) Error: Setup task error: gov.niarl.his.privacyca.TpmModule$TpmModuleException: TpmModule.setCredential returned nonzero error (19) Error: Cannot run RequestEndorsementCertificate: Setup error Error: Error from RequestAikCertificate: Configuration required Warning: Configuring RequestAikCertificate: Endorsement Certificate is missing Error: Error while executing setup: Encountered errors during setup`

tagent logs:

[logs.zip](https://github.com/opencit/opencit/files/341688/logs.zip)

I have read that some tpm chips do not come with the EC. I had to create de EK the first time in my TPM chip , so it seems to be my case.

How should I create the EC for the Trust agent to get installed correctly??

any help totally appreciated

Greetings.

**Comment by ZahediAquino** [comment]
Make sure you are using Trousers 0.3.13 or higher, clear the tpm, and try installation again.

**Comment by ryansavino** [comment]
The trustagent will create an EC for you if one is not present; this can be done only once, and it will register it with the opencit server.

@jemm14 is this a duplicate of #13 #10?

**Comment by jemm14** [comment]
yeah, I guess so. I will close them and stay with Issue #13
